packit_service/worker/testing_farm.py

```diff
@@ -93,6 +75,49 @@ def _trigger_payload(self, pipeline_id: str, chroot: str) -> dict:
             },
         }
 
+    def _payload(self, chroot: str) -> dict:
+        """
+        New Testing Farm API
+        https://testing-farm.gitlab.io/api/
+        """
+        compose, arch = self.get_compose_arch(chroot)
+        return {
+            "api_key": self.service_config.testing_farm_secret,
+            "test": {
+                "fmf": {
+                    "url": self.metadata.project_url,
+                    "ref": self.metadata.git_ref,
+                },
+            },
+            "environments": [
+                {
+                    "arch": arch,
+                    "os": {"compose": compose},
+                    "artifacts": [
+                        {
+                            "id": f"{self.job_owner}/{self.job_project}",
+                            "type": "fedora-copr-build",
+                        }
+                    ],
+                }
+            ],
+            "notification": {"webhook": f"{self.api_url}/testing-farm/results"},
+        }
+
+    def get_compose_arch(self, chroot) -> Tuple[str, str]:
+        # fedora-33-x86_64 -> Fedora-33, x86_64
+        compose, arch = chroot.rsplit("-", 1)
+        compose = compose.title()
```

Comment:
We might want to use some logic from aliases...  There are some weird names for that this will not work:
https://github.com/packit/packit/blob/c571c810bf633477cd470bb2652a4461cab8c1c0/tests/unit/test_config_aliases.py#L79

Reply:
Well, I can't tell how would a compose (which the new Testing Farm API uses) for this `opensuse-leap-15.0` be named, because currently [there are fedora composes only](https://api.dev.testing-farm.io/v0.1/composes), but I'd expect `Opensuse-Leap-15.0`, which is what my code does:

```python
compose, arch = "opensuse-leap-15.0-x86_64".rsplit("-", 1)
print(compose.title())
'Opensuse-Leap-15.0'
```

or maybe I just don't understand what you're suggesting?

Reply:
My bad. I've missed the `r` in `rsplit`....